fabfile/db/__init__.py

```diff
@@ -136,11 +161,11 @@ def init_db_sqlite(self, *args, **kwargs):
         """
         Init db to original state for sqlite.
         """
-        with lcd(self.base_path), settings(hide('warnings'), warn_only=True):
-            local('rm %s/mysite.db' % dj_settings.BASE_DIR)
-            local('%s/bin/python mysite/manage.py migrate' % env.venv_name)
-            local('%s/bin/python mysite/manage.py loaddata mysite/dumpdata/debug-wo-fsm.json' % env.venv_name)
-            local('%s/bin/python mysite/manage.py fsm_deploy' % env.venv_name)
+        with lcd(self.base_path+'/mysite/'), settings(hide('warnings'), warn_only=True):
+            local('rm %s' % (self.db_cfg['NAME']))
+            local('%s/bin/python manage.py migrate' % env.venv_name)
```

Comment:
In this point python interpreter is not accessible due to ``self.base_path+'/mysite/'``
Same below.